src/utils/eeprom/index.js

```diff
@@ -0,0 +1,93 @@
+#!/bin/env node
+
+const sleep = require('sleep-promise');
+const debug = require('debug')('eeprom');
+const util = require("util");
+const exec = util.promisify(require("child_process").exec);
+
+module.exports = class eeprom {
+
+    async info() {
+        try {
+            const serial = await this.readSerial();
+            const data = await this.parseSerial(serial);
+            debug(data);
+            return data;
+        } catch (error) {
+            throw error;
+        }
+    }
+
+    async writeSerial(serial) {
```

Comment:
I wonder if we should protect this behind a 'programming mode' flag. For example, you have to set a config var in order to be able to access this end point?